src/components/Feed/Offer/Info/StatusLabels/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FunctionComponent } from "react";
+import { useSeparatedStatusTranslations } from "$hooks";
+import { ApprovalStatus } from "$interfaces/ApprovalStatus";
+import { TargetApplicantType } from "$interfaces/Offer";
+import { StatusLabel } from "$components/StatusLabel";
+import classNames from "classnames";
+import styles from "./styles.module.scss";
+
+export const StatusLabels: FunctionComponent<IComponentProps> = (
+  {
+    extensionApprovalStatus,
+    graduadosApprovalStatus,
+    targetApplicantType
+  }
+) => {
+  const { graduados, extension } = useSeparatedStatusTranslations({
+    targetApplicantType,
+    graduadosApprovalStatus,
+    extensionApprovalStatus
+  });
+
+  return (
+    <div className={styles.separatedStatusLabel}>
+      {
+        extension &&
+        <StatusLabel
+          className={classNames({
+            [styles.extensionApprovalStatus]: graduados,
```

Comment:
ese class name lo llamaria algo mas como "is not only status

Reply:
por que? no entiendo, como is not only status ? es un status